src/api/libcellml/component.h

```diff
@@ -1,11 +1,24 @@
+/*
+Copyright 2015 University of Auckland
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.Some license of other
+*/
+
 #ifndef LIBCELLML_LIBCELLML_COMPONENT_H
 #define LIBCELLML_LIBCELLML_COMPONENT_H
 
-#include <string>
-#include <memory>
-
 #include "libcellml/libcellml_export.h"
```

Comment:
Fair enough, although a quick and dirty way to 'fix' it would be to rename the file after it has been generated by CMake. This would also allow to rename the file, if 'needed'. In other words, we could have a simple CMake macro that is used to generate those configuration files and rename them, if needed?